lib/presto-client/headers.js

```diff
@@ -1,45 +1,43 @@
-var Headers = exports.Headers = function(){
+var Headers = exports.Headers = {};
+var TrinoHeaders = exports.TrinoHeaders = {};
+
+// All values in this array are prefixed with X-Presto-/X-Trino-,
+// with the key being the value uppercased and dashes replaced
+// with underscores, e.g. `Max-Size` becomes
+// `{ MAX_SIZE: 'X-Presto-Max-Size' }` on Headers object.
+const commonPrefix = [
+  'User',
+  'Source',
+  'Catalog',
+  'Schema',
+  'Time-Zone',
+  'Current-State',
+  'Max-Wait',
+  'Max-Size',
+  'Page-Sequence-Id',
+  'Session',
+  'Prepared-Statement',
+];
+const legacyPrefix = {
+  'PREPARE': 'Prepared-Statement',
 };
-
-Headers.USER = 'X-Presto-User';
-
-Headers.SOURCE = 'X-Presto-Source';
-Headers.CATALOG = 'X-Presto-Catalog';
-Headers.SCHEMA = 'X-Presto-Schema';
-
-Headers.TIME_ZONE = 'X-Presto-Time-Zone';
-
-Headers.CURRENT_STATE = 'X-Presto-Current-State';
-Headers.MAX_WAIT = 'X-Presto-Max-Wait';
-Headers.MAX_SIZE = 'X-Presto-Max-Size';
-Headers.PAGE_SEQUENCE_ID = 'X-Presto-Page-Sequence-Id';
-
-Headers.SESSION = 'X-Presto-Session';
-Headers.PREPARE = 'X-Presto-Prepared-Statement';
-
-Headers.USER_AGENT = 'User-Agent';
-
-Headers.AUTHORIZATION = 'Authorization';
-
-var TrinoHeaders = exports.TrinoHeaders = function(){
-};
-
-TrinoHeaders.USER = 'X-Trino-User';
-
-TrinoHeaders.SOURCE = 'X-Trino-Source';
-TrinoHeaders.CATALOG = 'X-Trino-Catalog';
-TrinoHeaders.SCHEMA = 'X-Trino-Schema';
-
-TrinoHeaders.TIME_ZONE = 'X-Trino-Time-Zone';
-
-TrinoHeaders.CURRENT_STATE = 'X-Trino-Current-State';
-TrinoHeaders.MAX_WAIT = 'X-Trino-Max-Wait';
-TrinoHeaders.MAX_SIZE = 'X-Trino-Max-Size';
-TrinoHeaders.PAGE_SEQUENCE_ID = 'X-Trino-Page-Sequence-Id';
-
-TrinoHeaders.SESSION = 'X-Trino-Session';
-TrinoHeaders.PREPARE = 'X-Trino-Prepared-Statement';
-
-TrinoHeaders.USER_AGENT = 'User-Agent';
-
-TrinoHeaders.AUTHORIZATION = 'Authorization';
+const common = [
+  'User-Agent',
+  'Authorization',
+];
+
+const transform = (key) => {
```

Comment:
Make this name understandable. For example: `transformToHeaderKeyName`

Reply:
:+1: Done in 1d0cc0c77c896f058f7eebd9afd9faab4cf3b21c